lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -412,10 +417,8 @@ void _testVerticalScrolling() {
 
     semantics().semanticsEnabled = false;
   },
-      // TODO(nurhan): https://github.com/flutter/flutter/issues/50590
       // TODO(nurhan): https://github.com/flutter/flutter/issues/50754
-      skip: browserEngine == BrowserEngine.webkit ||
-          browserEngine == BrowserEngine.edge);
+      skip: browserEngine == BrowserEngine.edge);
```

Comment:
This file isn't running on edge anyway.

Reply:
That's true. I think I'll create another PR for removing all edge requirements though. Since our Edge testing code is for legacy edge. If we change it to new Edge all should pass (I hope :) )